source/documentation/errors.html.md.erb

```diff
@@ -75,172 +75,172 @@ All errors from TGP Digital will return a JSON map with two keys: "code" and "me
-    <td>400</td>
+    <td>Trying to create or update a record with a duplicate traderRef</td>
   </tr>
   <tr>
     <td>011</td>
     <td>comcode</td>
-    <td>400</td>
+    <td>Mandatory field comcode was missing from body</td>
   </tr>
   <tr>
     <td>012</td>
     <td>goodsDescription</td>
-    <td>400</td>
+    <td>Mandatory field goodsDescription was missing from body</td>
   </tr>
   <tr>
     <td>013</td>
     <td>countryOfOrigin</td>
-    <td>400</td>
+    <td>Mandatory field countryOfOrigin was missing from body</td>
   </tr>
   <tr>
     <td>014</td>
     <td>category</td>
-    <td>400</td>
+    <td>Mandatory field category was missing from body</td>
   </tr>
   <tr>
     <td>015</td>
     <td>assessments[].assessmentId</td>
-    <td>400</td>
+    <td>Optional field assessmentId is in the wrong format</td>
   </tr>
   <tr>
     <td>016</td>
     <td>assessments[].primaryCategory</td>
-    <td>400</td>
+    <td>Optional field primaryCategory is in the wrong format</td>
   </tr>
   <tr>
     <td>017</td>
     <td>assessments[].primaryCategory.condition.type</td>
-    <td>400</td>
+    <td>Optional field type is in the wrong format</td>
   </tr>
   <tr>
     <td>018</td>
     <td>assessments[].primaryCategory.condition.conditionId</td>
-    <td>400</td>
+    <td>Optional field conditionId is in the wrong format</td>
   </tr>
   <tr>
     <td>019</td>
     <td>assessments[].primaryCategory.condition.conditionDescription</td>
-    <td>400</td>
+    <td>Optional field conditionDescription is in the wrong format</td>
   </tr>
   <tr>
     <td>020</td>
     <td>assessments[].primaryCategory.condition.conditionTraderText</td>
-    <td>400</td>
+    <td>Optional field conditionTraderText is in the wrong format</td>
   </tr>
   <tr>
     <td>021</td>
     <td>supplementaryUnit</td>
-    <td>400</td>
+    <td>Optional field supplementaryUnit is in the wrong format</td>
   </tr>
   <tr>
     <td>022</td>
     <td>measurementUnit</td>
-    <td>400</td>
+    <td>Optional field measurementUnit is in the wrong format</td>
   </tr>
   <tr>
     <td>023</td>
     <td>comcodeEffectiveFromDate</td>
-    <td>400</td>
+    <td>Mandatory field comcodeEffectiveFromDate was missing from body</td>
   </tr>
   <tr>
     <td>024</td>
     <td>comcodeEffectiveToDate</td>
-    <td>400</td>
+    <td>Optional field comcodeEffectiveToDate is in the wrong format</td>
   </tr>
   <tr>
     <td>025</td>
     <td>recordId</td>
-    <td>400</td>
+    <td>The recordId has been provided in the wrong format</td>
   </tr>
   <tr>
     <td>026</td>
     <td>recordId</td>
-    <td>400</td>
+    <td>The requested recordId to update doesn’t exist</td>
   </tr>
   <tr>
     <td>027</td>
     <td>accreditationStatus</td>
-    <td>400</td>
+    <td>There is an ongoing accreditation request and the record can not be updated</td>
   </tr>
   <tr>
     <td>028</td>
     <td>lastUpdatedDate</td>
-    <td>400</td>
+    <td>This record has been deactivated and cannot be updated</td>
```

Comment:
This isn't the correct information (although the Confluence table is wrong so I can see why it ended up here)